types/node-fetch/index.d.ts

```diff
@@ -1,110 +1,174 @@
-// Type definitions for node-fetch v1.6
+// Type definitions for node-fetch v2.1.2
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
@DanielRosenwasser thanks for taking the time to review. I've enabled the linting rules and addressed the shortcomings.